content/en/contribute/code/core/dev-environment.md

```diff
@@ -143,7 +156,7 @@ Create a `docker-compose.yml` and `couchdb-override.yml` files under the `~/cht-
 
 ```
 mkdir -p ~/cht-docker
-curl -s -o ~/cht-docker/docker-compose.yml https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:4.1.0/docker-compose/cht-couchdb.yml
+curl -s -o ~/cht-docker/docker-compose.yml https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:master/docker-compose/cht-couchdb.yml
```

Comment:
we'd wanted to folks to use a stable branch in other docs, so intentionally changed compose URLs from from `master` -> `4.1.0`.  I ...think this is OK though b/c this is core devs who should be using master?

@garethbowen  - thoughts?

Reply:
I think there are three options... hardcoding a version, using master, or leaving an undefined variable in there that must be replaced. I don't like hardcoding a version (4.1.0) because it'll have to be updated every couple of months.

I think `master` is best in this context, but would be happy to replace with `${version}` instead, which sort of teaches people how to install legacy versions for developing against old versions.